_includes/docs/user-guide/alarms.md

```diff
@@ -56,72 +56,169 @@ Assuming the following sequence of events:
 
 Hence, you should create a single "HighTemperature" alarm with start time = 12:30 and end time = 13:00.
 
-##### Propagation
+Now that you know the theory, let's proceed to practical tutorials.
```

Comment:
Why Propagation was removed?

Reply:
You can't remove the propagation section even if you mention it in the end of this page